src/main/frontend/db_worker.cljs

```diff
@@ -200,11 +196,11 @@
 
 (defn- get-search-db
   [repo]
-  (get-sqlite-conn repo {:search? true}))
+  (state/get-sqlite-conn repo {:search? true}))
 
 
 #_:clj-kondo/ignore
-(defclass SQLiteDB
+(defclass DBWorker
```

Comment:
`SQLiteDB` was renamed to `DBWorker` because it exposes other functions such as rtc-related to the main thread.